src/steadystateproblem.cpp

```diff
@@ -26,19 +26,30 @@ SteadystateProblem::SteadystateProblem(const Solver &solver, const Model &model)
       sx_(model.nx_solver, model.nplist()), sdx_(model.nx_solver, model.nplist()),
       xB_(model.nJ * model.nx_solver), xQ_(model.nJ * model.nx_solver),
       xQB_(model.nplist()), xQBdot_(model.nplist()),
-      dJydx_(model.nJ * model.nx_solver * model.nt(), 0.0) {
+      dJydx_(model.nJ * model.nx_solver * model.nt(), 0.0),
+      steady_state_status_(3, SteadyStateStatus::not_run) {
           /* maxSteps must be adapted if iterative linear solvers are used */
           if (solver.getLinearSolver() == LinearSolver::SPBCG) {
               max_steps_ = solver.getNewtonMaxSteps();
               numlinsteps_.resize(2 * max_steps_, 0);
           }
+          if (solver.getSensitivityOrder() == SensitivityOrder::none)
+              return;
           /* Check for compatibility of options */
           if (solver.getSensitivityMethod() == SensitivityMethod::forward &&
-              solver.getSensitivityMethodPreequilibration() == SensitivityMethod::adjoint &&
-              solver.getSensitivityOrder() > SensitivityOrder::none)
+              solver.getSensitivityMethodPreequilibration()
+                  == SensitivityMethod::adjoint)
               throw AmiException("Preequilibration using adjoint sensitivities "
                                  "is not compatible with using forward "
                                  "sensitivities during simulation");
+          
+          if (solver.getSensitivityMethodPreequilibration()
+                  == SensitivityMethod::adjoint &&
+              model.getSteadyStateSensitivityMode() ==
+                  SteadyStateSensitivityMode::simulationFSA)
+              throw AmiException("Preequilibration using adjoint sensitivities "
+                                 "is not compatible with simulationFSA "
+                                 "sensitivity mode.");
```

Comment:
> I understand that setting `SteadyStateSensitivityMode::simulationFSA` and allowing to use adjoint preequilibration is counter-intuitive. However, it would make much sense to allow the user to switch off the Newton solver and using adjoint preequilibration... Would be happy if there was a way to encode this...

How about setting the allowed newton steps to 0?

Reply:
Ideally, there would be the options newtonOnly, simulationFSA, and simulationASA and something that first tries Newton and uses simulation only if Newton fails... But no need to implement this now.

Something else: I'd advocate to move the steadyStateSensitivitiyMode from model to solver. The value may be model dpeendent, but it encodes the behavior of the solver. Therefore, I'd be happy if we could move this at some point. What's your opinion on this?

Reply:
> Ideally, there would be the options newtonOnly, simulationFSA, and simulationASA and something that first tries Newton and uses simulation only if Newton fails... But no need to implement this now.

I think most of these settings can already be achieved with existing options.

> Something else: I'd advocate to move the steadyStateSensitivitiyMode from model to solver. The value may be model dpeendent, but it encodes the behavior of the solver. Therefore, I'd be happy if we could move this at some point. What's your opinion on this?

Yes, I agree, I found it confusing that this is in model, not solver.